This patch was written in-house and was intended for contribution upstream,
but upstream already has accepted a fix from another contributor in 2.3:
https://github.com/tmux/tmux/commit/3ed0973493862a3720c8dd46ee6d58f3d15f2490

--- tmux-2.2/osdep-sunos.c.orig	2015-04-25 13:44:57.000000000 -0700
+++ tmux-2.2/osdep-sunos.c	2016-08-25 08:56:09.437920629 -0700
@@ -18,6 +18,7 @@
 
 #include <sys/types.h>
 #include <sys/stat.h>
+#include <sys/param.h>
 
 #include <event.h>
 #include <fcntl.h>
